### ISSUE TITLE
Localization support

### ISSUE THREAD

**Issue by Fevol** [Fevol/obsidian-translate#10]
- [ ] Replace every English string with corresponding localization identifier

**Comment by Fevol** [comment]
Many thanks for reaching out @andremeireles!

Aside from translating languages, the localization system for settings/commands/etc. has not been set-up yet. I will not have the time to implement that for the next two weeks, so timeline for the feature should be a month at most. However, if you've added translations of the languages for `pt-br`, I can absolutely implement those quickly, since that system already works.

**Comment by Fevol** [comment]
Hopefully it's not too late, but I just realized that you might be able to reuse the majority of the translations for `pt-BR` from `pt`. I don't speak the language, but I assume there is significant overlap between Portuguese and Brazilian Portuguese?

**Comment by andremeireles** [comment]
You're right, it's not too late, it's possible to reuse the **pt** list with minor modifications.

I will be waiting for your return and the implementations and as soon as they are available, I will sent you the UI strings because I got really excited about them, so I wanted to prioritize that instead the list of languages.
